docs/faq.md

```diff
@@ -13,6 +13,10 @@ the output of a failed installation. Usually, the output points pretty clearly
 at the source of the problem. Watchmaker can be re-installed over itself with
 no problem, so once the root cause is resolved, simply re-install watchmaker.
 
+## Why does the watchmaker install fail if my system is FIPS enabled?
+
+This is primarily a question for Red Hat (and derived distributions). As of this writing, the `pip` utility in all Red Hat releases up through 7.4.1708, default to looking for pypi packages signed with MD5 signatures. If you've enabled FIPS (or are using a build that has FIPS pre-enabled), MD5 is disabled in the kernel (due to being a weak hashing-method). You can either disable FIPS (not recommended) or explicitly force `pip` to use a different signature-index. The latter is detailed in the Linux section of the :doc:`usage <usage#linux>` document.
```

Comment:
Unfortunately it seems can't currently link to an anchor directly, the way the docs are generated. Need to just do `usage <usage>` for now.

Reply:
Blergh. That's sucktacular. :-1:

Reply:
it's just the way it's rendered from markdown to rst to html. md -> html, no prob. or rst -> html, no prob. but md -> rst is a bit wonky. the recommonmark package from read-the-docs that currently handles the conversion has not been getting a lot of love for a while now.